ssl/statem/statem_lib.c

```diff
@@ -272,7 +272,9 @@ int tls_construct_cert_verify(SSL *s, WPACKET *pkt)
         goto err;
     }
 
-    if (EVP_DigestSignInit(mctx, &pctx, md, NULL, pkey) <= 0) {
+    if (EVP_DigestSignInit_ex(mctx, &pctx,
+                              md == NULL ? NULL : EVP_MD_name(md),
```

Comment:
Did we think about having `EVP_MD_name()` accept `NULL` and return `NULL`?  This ternary appears a few times already...

Reply:
Hmmmm.....maybe......not sure. But I'd say out of scope for this PR.